robottelo.properties.sample

```diff
@@ -145,6 +145,8 @@ ssh_key=
 # password=
 # basedn=
 # grpbasedn=
+# realm=
+# nameserver=
```

Comment:
GitLab MR is to be done for the same right?

Reply:
yeah will be adding that